app/controllers/animals_controller.rb

```diff
@@ -0,0 +1,45 @@
+class AnimalsController < ApplicationController
+
+    def index
+        animals = Animal.all
+        render json: animals
+    end
+
+    def show
+        animal = Animal.find(params[:id])
+        render :json => animal.to_json(:include => :sightings)
+    end
+
+    def create
+        animal = Animal.create (animal_params)
+        if animal.valid?
+            render json: animal
+        else 
+            render json: animal.errors
+        end
+    end
+
+    def destroy
+        animal = Animal.find(params[:id])
+        if animal.destroy
+            render json:animal
+        else 
+            render json:errors
+        end
+    end
+
+    def update
```

Comment:
Story 6: Story: As the consumer of the API I can update an animal sighting in the database.

The code is definitely correct. The animal params method below is correct as well. Which means this whole story is definitely working and done correct. Nice job!